crates/binjs_es6/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![feature(box_patterns)]
 #![recursion_limit="128"] // We have deeply nested data structures...
 
+#[macro_use]
```

Comment:
I put the above macro inside crates/binjs_io/src/io.rs, since I cannot find any other place to put it, to make it available everywhere which needs it.
let me know if you know better place :)
(I'm rust beginner and I think I'm missing something)

Reply:
If you're using it both in `binjs_generate_library` and at runtime, it should go in `binjs_shared`. If it's only runtime, `binjs_io` is good.

I should write a README.md explaining this :)